tests/integration/Tribe/Events/Integrations/QR/ControllerTest.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace Tribe\Events\Integrations\QR;
+
+/**
+ * Tests QR Controller functionality
+ *
+ * @group   core
+ * @group   qr
+ *
+ * @package TribeEvents
+ */
+class ControllerTest extends \Codeception\TestCase\WPTestCase {
+
+	/**
+	 * @var \TEC\Events\QR\Controller
+	 */
+	protected $controller;
+
+	function setUp() {
+		parent::setUp();
+		$this->controller = tribe(\TEC\Events\QR\Controller::class);
```

Comment:
This file (and other test files) should still follow proper spacing even though we don't run PHPCS on it.

```suggestion
		$this->controller = tribe( \TEC\Events\QR\Controller::class );
```